off_your_rocker/bind.py

```diff
@@ -0,0 +1,47 @@
+import os
+from rocker.extensions import RockerExtension
+
+
+class Bind(RockerExtension):
+
+    name = 'oyr_bind'
+
+    @classmethod
+    def get_name(cls):
+        return cls.name
+
+    def precondition_environment(self, cli_args):
+        pass
+
+    def validate_environment(self, cli_args):
+        pass
+
+    def get_preamble(self, cli_args):
+        return ''
+
+    def get_snippet(self, cli_args):
+        return ''
+
+    def get_docker_args(self, cli_args):
+        args = ['']
+        print(cli_args)
+        for mount in cli_args['oyr_bind']:
+            for d in mount.split(" "):
+                if ":" in d:
+                    src,dst = d.split(":")
+                else:
+                    src = d
+                    dst = d
+                src = os.path.abspath(src)
+                args.append(f'--mount type=bind,source={src},target={dst}')
```

Comment:
As mentioned in my other comment, I believe that using a volume over a bind mount is preferable.

Reply:
Also, if it's preferable to use a volume over a bind mount, is there any way we can just add the flexibility to the `mount` flag to handle user-specified container paths instead of adding a whole new `bind` argument? That would be simpler, and remove any potential confusion of the difference between the `mount` and `bind` arguments.